Client/src/slices/loginSlice.js

```diff
@@ -3,7 +3,6 @@ import axios from "axios";
 import { USER_SERVICE_URL } from "../constants/servicesURLs";
```

Comment:
Seems 2 different folders appear again: client/src and Client/src in PR. Please check it